app/javascript/dashboard/components-next/HelpCenter/EmptyState/Article/ArticleEmptyState.vue

```diff
@@ -0,0 +1,97 @@
+<script setup>
+// import { ref } from 'vue';
+import EmptyStateLayout from 'dashboard/components-next/EmptyStateLayout.vue';
+import Button from 'dashboard/components-next/button/Button.vue';
+import ArticleCard from 'dashboard/components-next/HelpCenter/ArticleCard/ArticleCard.vue';
+// import AddLocaleDialog from 'dashboard/playground/HelpCenter/components/AddLocaleDialog.vue';
+
+const articles = [
+  {
+    title: "How to get an SSL certificate for your Help Center's custom domain",
+    status: 'draft',
+    updatedAt: '2 days ago',
+    author: 'Michael',
+    category: '⚡️ Marketing',
+    views: 3400,
+  },
+  {
+    title: 'Setting up your first Help Center portal',
+    status: '',
+    updatedAt: '1 week ago',
+    author: 'John',
+    category: '🛠️ Development',
+    views: 400,
+  },
+  {
+    title: 'Best practices for organizing your Help Center content',
+    status: 'archived',
+    updatedAt: '3 days ago',
+    author: 'Fernando',
+    category: '💰 Finance',
+    views: 400,
+  },
+  {
+    title: 'Customizing the appearance of your Help Center',
+    status: '',
+    updatedAt: '5 days ago',
+    author: 'Jane',
+    category: '💰 Finance',
+    views: 400,
+  },
+];
+
+// const addLocaleDialogRef = ref(null);
+// const openDialog = () => {
+//   addLocaleDialogRef.value.dialogRef.open();
+// };
+// const handleDialogConfirm = () => {
+//   // Add logic to create a new portal
+// };
+</script>
+
+<!-- TODO: Add i18n -->
+<!-- eslint-disable vue/no-bare-strings-in-template -->
+<template>
+  <EmptyStateLayout>
+    <template #empty-state-item>
+      <div class="grid grid-cols-1 gap-4">
+        <ArticleCard
+          v-for="(article, index) in articles"
+          :key="`article-${index}`"
+          :title="article.title"
+          :status="article.status"
+          :updated-at="article.updatedAt"
+          :author="article.author"
+          :category="article.category"
+          :views="article.views"
+        />
+      </div>
+    </template>
+    <template #empty-state>
+      <div class="flex flex-col items-center justify-center gap-6">
+        <div class="flex flex-col items-center justify-center gap-2">
+          <h2
+            class="text-3xl font-medium text-center text-slate-900 dark:text-white"
+          >
+            Write an article
+          </h2>
+          <p
+            class="max-w-lg text-base text-center text-slate-600 dark:text-slate-300"
+          >
+            Write a rich article, let’s get started!
+          </p>
```

Comment:
@iamsivin I see this usage in the portal empty state too, ideally the title and the subtitle should be exposed as both a prop and a slot. 

```vue
<EmptyStateLayout>
   <template #empty-state-item>
   </template>
   <template #empty-state>
       <template #title> Hello World </template>
       <template #subtitle> This is a lot </template>
       <template #actions> ... </template>
   </template>
</EmptyStateLayout>
```

The `title` and `subtitle` should be a prop and the actions can be added as a slot.
That way, the developer does not have to worry about styling the title or the subtitle. and the layout. In your current implementation the `EmptyStateLayout` does too little to a point of being futile. The default should be a strict with slots to allow flexibility.

Reply:
@scmmishra Updated the PR's with the changes you suggested. Please take a look.